### PR TITLE
[Hived]: Add config quick start docs

### PR DESCRIPTION
**Pull request by yqwang-ms**
Resolves https://github.com/microsoft/pai/issues/4092

### CODE REVIEW

File: subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    1. It is like the [Azure VM Series](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-gpu) or [GCP Machine Types](https://cloud.google.com/compute/docs/machine-types).
+    2. Currently, the `gpuTypes` is not directly used by HivedScheduler, but it is used by [OpenPAI RestServer](https://github.com/microsoft/pai/tree/master/src/rest-server) to setup proportional Pod resource requests and limits. So, if you are not using with [OpenPAI RestServer](https://github.com/microsoft/pai/tree/master/src/rest-server), you can skip to config it.
+
+    **Example:**
+
+    Assume you have some `K80` nodes of the same SKU in your cluster, and you want to schedule Pods on them:
+
+    1. Using `kubectl describe nodes` to check if these `K80` nodes have nearly the same [Allocatable Resources](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources), especially for gpu, cpu, memory. If not, please fix it. Assume the aligned resources are: 4 gpus, 23 cpus, and 219GB memory.
+
+    2. Then proportionally, each gpu request should also has ceil(23/4)=5 cpus and ceil(219/4)=54GB memory along with it, so config the `K80` `gpuType` as below:
```
Comment (fanyangCS) [inline subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md]: shouldn't it be floor(23/4)?

File: subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            virtualCells:
+            - cellType: K80-NODE-POOL.K80-NODE
+              cellNumber: 2
+        ```
+        Notes:
+        1. The name of `virtualCluster` should be constrained by the [K8S naming convention](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names).
+        2. The `virtualCells.cellType` should be full qualified and should be started with a `cellType` which is explicitly referred in `physicalCells`.
+
+
+### <a name="ConfigDetail">Config Detail</a>
```
Comment (fanyangCS) [inline subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md]: can you give a real example of  <your_physical_clusters_config> and <your_virtual_clusters_config> here?

File: subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md (hunk 1 above)
Comment (yqwang-ms) [reply subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md]: Good catch my typo. Will fix in next pr

File: subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md (hunk 2 above)
Comment (yqwang-ms) [reply subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md]: Do you mean merge above yaml pieces into a whole one?

Comment (fanyangCS) [reply subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md]: yes. a complete yaml file so people could put these pieces into one complete file they need to prepare.

Comment (yqwang-ms) [reply subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md]: Done in https://github.com/microsoft/pai/pull/4102/files

**Pull request merged**